packages/bitbucket-v8-coverage-report/index.ts

```diff
@@ -60,18 +75,25 @@ async function uploadReport() {
       branches: coverageResults.total.branches.pct,
     };
 
-    const reportResult = getReportResult(coverageResultPercentage, {
-      statements: minStatementsCoverage,
-      lines: minLinesCoverage,
-      functions: minFunctionsCoverage,
-      branches: minBranchesCoverage,
-    });
+    const averageCoverage = Math.round(Object.values(coverageResultPercentage).reduce((acc, val) => acc + val, 0) / 4);
```

Comment:
4 is hardcoded, but I don't think it will change anytime soon, so perhaps it is fine

Reply:
ok